mod_janus.c

```diff
@@ -113,13 +113,20 @@ switch_status_t joined(janus_id_t serverId, janus_id_t senderId, janus_id_t room
 	switch_channel_t *channel;
 	private_t *tech_pvt;
 	server_t *pServer;
+	char * uuid;
```

Comment:
You didn't initialize` uuid` to NULL here. I know it doesn't impact code as it was initialized in the if condition below. But I noticed it was initialized in the other subsequent functions.